src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -73,25 +73,20 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         });
     }
 
-    private _getParamsScript(initializationData: PaypalCommerceInitializationData, cart: Cart, isMessagesAvailable: boolean): PaypalCommerceScriptParams {
+    private _getParamsScript(initializationData: PaypalCommerceInitializationData, cart: Cart): PaypalCommerceScriptParams {
         const { clientId, intent, isPayPalCreditAvailable, merchantId } = initializationData;
         const disableFunding: DisableFundingType = [ 'card' ];
-        const components: ComponentsScriptType = ['buttons'];
 
         if (!isPayPalCreditAvailable) {
             disableFunding.push('credit');
         }
 
-        if (isMessagesAvailable) {
-            components.push('messages');
-        }
-
         return {
             'client-id': clientId,
             'merchant-id': merchantId,
             commit: false,
             currency: cart.currency.code,
-            components,
+            components: ['buttons', 'messages'],
```

Comment:
What happens if `messagingContainer` is not passed in? At the moment, it seems you can actually not pass in `messagingContainer` (which is why you still have `isMessagesAvailable` check).

Reply:
Nothing happens. 
Values for `components` needs for PayPal script. After load script we call `payPal.Buttons.render` for rendering smart payment buttons and call `payPal.Messages.render` for rendering credit banners. 
In this case we add always `messages` to `components` to avoid conflict with script [here](https://github.com/bigcommerce/content/pull/615)

Reply:
Ok cool, I guess what you mean is unless `payPal.Messages.render` is called, nothing happens, even if `components` contains `messages`.